microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md

```diff
@@ -1,25 +1,78 @@
+title: Windowed vs. Visual hosting of WebView2
+description: Deciding whether to have your app use Windowed hosting vs. Visual hosting of the WebView2 control.  Hosting WebView2 in windowed or visual environments.
 author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.service: microsoft-edge
 ms.subservice: webview
-ms.date: 10/24/2022
+ms.date: 09/18/2024
 ---
-# Windowed vs. visual hosting of WebView2
+# Windowed vs. Visual hosting of WebView2
 
-There are two options for hosting the Microsoft Edge WebView2 control in your app: windowed hosting and visual hosting.
+<!-- outline Sep 28, 2024:
+=======================================
+## Scenarios for selecting the hosting approach
+#### Windowed hosting: For displaying content quickly and easily
+###### Advantages
+###### Disadvantages
+
+#### Window to Visual hosting: Similar experience as Windowed hosting with added benefits and tradeoff
+###### Advantages
+###### Disadvantages
+
+#### Visual hosting: For more granular control over layout
+###### Advantages and disadvantages
 
-If you use windowed hosting, which is a good starting point for most apps, you don't need to read this article.  If you want to provide a more custom user experience (UX), and want to use visual hosting, read this article.
+=======================================
+## Compatibility and constraints
+#### Operating systems
+#### Rendering WebView2 in framework and non-framework apps    [APIs tabset]
+
+=======================================
+## Windowed hosting and Window to Visual hosting
+#### Benefits of Windowed hosting
+#### Benefits of Window to Visual hosting
+#### APIs for Windowed hosting and Window to Visual hosting
+#### Window management                            [APIs tabset]
+###### Sizing, positioning, and visibility        [APIs tabset]
+###### Zooming                                    [APIs tabset]
+#### Rasterization scale                          [APIs tabset] err? same level as surrounding headings, in "Overview of APIs"
+###### Focus and tabbing                          [APIs tabset]
+###### Parent window                              [APIs tabset]
+###### Keyboard accelerators                      [APIs tabset]
+###### Default background color                   [APIs tabset]
+
+=======================================
+## Visual hosting
+#### APIs for Visual hosting
+#### Composition-based rendering                  [APIs tabset]
+#### Output                                       [APIs tabset]
+#### Input                                        [APIs tabset]
+#### Accessibility                                [APIs tabset]
+
+=======================================
+## See also
+
+Overview of APIs: https://learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis
+-->
+
+There are three options for hosting the Microsoft Edge WebView2 control in your app:
+* Windowed hosting.
+* Window to Visual hosting.
+* Visual hosting.
```

Comment:
```suggestion
* The Windowed hosting mode.
* The Window to Visual hosting mode.
* The Visual hosting mode.
```
@johna-ms re: term'y, capz'n

Reply:
i'm going to make this change to press this point and send for review b/c "mode" is heavily used in article